src/NUnitEngine/nunit.engine/Services/ExtensionService.cs

```diff
@@ -432,7 +446,7 @@ public void FindExtensionsInAssembly(ExtensionAssembly assembly)
                 if (versionArg != null && new Version((string)versionArg) > ENGINE_VERSION)
                     continue;
 
-                var node = new ExtensionNode(assembly.FilePath, type.FullName);
+                var node = new ExtensionNode(assembly.FilePath, type.FullName, assemblyTargetFramework);
```

Comment:
Once we decide that we can load the assembly in the current runtime, why is the assemblyTargetFramework still needed in the extension node? Is it only so that the console runner can display it?

Reply:
Yes, exactly. Info is only stored in `ExtensionNode` to be accessible to runners.